packages/wdio-selenium-standalone-service/src/launcher.ts

```diff
@@ -1,5 +1,8 @@
 import logger from '@wdio/logger'
 
+declare module 'util' {
+    function promisify<T1, TResult>(fn: (arg1: T1, callback: (err: any, result: TResult) => void) => void): (arg1: T1) => Promise<TResult>;
+}
```

Comment:
Why is this necessary?

Reply:
There seems to be some issue with promisify overloading , similar issue is mentioned here :
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/20497

By using the above code, we are specifying which method to overload instead of the generic one